pkg/cmd/recoverycontroller/cmd.go

```diff
@@ -90,6 +90,7 @@ func (o *Options) Run(ctx context.Context) error {
 	}
 
 	resourceSyncController := resourcesynccontroller.NewResourceSyncController(
+		"KubeSchedulerOperator",
```

Comment:
same issue here: https://github.com/openshift/cluster-kube-apiserver-operator/pull/1722#discussion_r1777595279

Reply:
seems that we go with the operand name: https://github.com/openshift/cluster-kube-apiserver-operator/commit/6f3ed3bc9b991818a5203c387bed69c32387fd0b